spec/services/accept_unconditional_offer_spec.rb

```diff
@@ -75,5 +75,15 @@
       expect(ActionMailer::Base.deliveries.first.to).to eq [application_choice.application_form.candidate.email_address]
       expect(ActionMailer::Base.deliveries.first.subject).to match(/You’ve accepted/)
     end
+
+    it 'sends an email to the candidates referees' do
+      FeatureFlag.activate(:new_references_flow)
+      application_form = create(:application_form, application_references: [create(:reference, :not_requested_yet), create(:reference, :not_requested_yet)])
+      application_choice = create(:application_choice, status: :offer, application_form: application_form)
+
+      expect { described_class.new(application_choice:).save! }.to change { ActionMailer::Base.deliveries.count }.by(3)
+      expect(ActionMailer::Base.deliveries.first.to).to eq [application_choice.application_form.application_references.first.email_address]
```

Comment:
will it always be the first, or are we potentially introducing a flaky test here?

Reply:
I tried
`for i in `seq 10`; do be rspec spec/services/accept_unconditional_offer_spec.rb:79 ; done `
to check this and was green each time 🤷‍♂️